packages/dd-trace/src/telemetry/index.js

```diff
@@ -6,10 +6,6 @@ const os = require('os')
 const dependencies = require('./dependencies')
 const { sendData } = require('./send-data')
 
-const HEARTBEAT_INTERVAL = process.env.DD_TELEMETRY_HEARTBEAT_INTERVAL
```

Comment:
Why was this moved? It doesn't seem to support programmatic config or be used anywhere else.

Reply:
I moved it to group all the telemetry properties reading in one place.